streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -76,6 +76,25 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    public void shouldNotThrowExceptionOnRestoreWithBulkLoad() throws Exception {
```

Comment:
I don't think this test proves that the fix works, i.e, it passes if i comment out lines 173 -> 176 in RocksDBStore